core/src/main/java/brooklyn/entity/basic/EntityDynamicType.java

```diff
@@ -130,17 +130,38 @@ public void addEffector(Effector<?> newEffector) {
             instance.emit(AbstractEntity.EFFECTOR_ADDED, newEffector.getName());
     }
 
-    /** Adds an effector with an explicit body */
+    /**
+     * Adds an effector with an explicit body to this entity.
+     */
     @Beta
     public <T> void addEffector(Effector<T> effector, EffectorTaskFactory<T> body) {
         addEffector(new EffectorAndBody<T>(effector, body));
     }
-    /** Adds an effector with an explicit body */
+
+    /**
+     * Adds an effector with an explicit body to this entity.
+     */
     @Beta
     public <T> void addEffector(Effector<T> effector, EffectorBody<T> body) {
         addEffector(effector, new EffectorBodyTaskFactory<T>(body));
     }
 
+    /**
+     * Removes the given {@link Effector} from this entity.
+     * <p>
+     * Note that if the argument is an instance of {@link EffectorWithBody} it will
+     * still be possible to invoke the effector on the entity by calling
+     * <code>entity.invoke(effector, argumentsMap)</code>.
+     */
```

Comment:
The ability to invoke removed efectors is a bug, this comment should reflect that. Is it trivial to fix `entity.invoke()`?